docker.sh

```diff
@@ -21,9 +20,15 @@ fi
 # replace slashes with underscore
 GIT_BRANCH=${GIT_BRANCH/\//_}
 
+VERSION=$(cat VERSION.txt)
 echo "$VERSION"
 
-docker build --build-arg version="$VERSION" --ssh default -t "$IMAGE_NAME:$GIT_BRANCH" .
+docker build \
+  --label org.opencontainers.image.version="${VERSION}" \
+  --label org.opencontainers.image.created="$(date -Iseconds)" \
+  --label org.opencontainers.image.revision="${CI_COMMIT_SHORT_SHA}" \
```

Comment:
I think this value is only set for gitlab ci, did you mean to use `${COMMIT}`

Reply:
changed to use git to get commit hash. So it will be independent of the pipeline.